addons/mail/models/mail_message_reaction.py

```diff
@@ -22,3 +23,16 @@ def init(self):
     _sql_constraints = [
         ("partner_or_guest_exists", "CHECK((partner_id IS NOT NULL AND guest_id IS NULL) OR (partner_id IS NULL AND guest_id IS NOT NULL))", "A message reaction must be from a partner or from a guest."),
     ]
+
+    def _reaction_groups(self, message):
```

Comment:
The method should be rewritten to use `self` as reactions, not taking message as param

Reply:
We need the `message.id` in line 36. So what can we do without message as a param?

Reply:
You have `message_id` on each reaction, you can group them together by `message_id`.